src/ui/components/SecondaryToolbox/react-devtools/utils/findProtocolObjectProperty.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  NamedValue,
+  Object as ProtocolObject,
+  Property as ProtocolProperty,
+} from "@replayio/protocol";
+
+export function findProtocolObjectProperty(
```

Comment:
:speech_balloon: I hate that we have to do this :)

Reply:
You and me both